registration-processor/pre-processor/registration-processor-osi-validator-stage/src/main/java/io/mosip/registration/processor/stages/osivalidator/OSIValidator.java

```diff
@@ -613,6 +620,11 @@ private boolean validateUserBiometric(String registrationId, String userId, List
 			String individualType, InternalRegistrationStatusDto registrationStatusDto)
 			throws ApisResourceAccessException, IOException, BioTypeException, AuthSystemException, CertificateException, NoSuchAlgorithmException {
 
+		if(INDIVIDUAL_TYPE_USERID.equalsIgnoreCase(individualType)) {
+			userId = getVidByUserId(userId);
```

Comment:
Rename this method to getIndividualIdByUserId

Reply:
Renamed method name to getIndividualIdByUserId